src/components/Identity/customerIds/createCustomerIds.js

```diff
@@ -6,7 +6,7 @@ import {
   clone
 } from "../../../utils";
 import { CUSTOMER_ID_HASH } from "../constants/cookieNames";
-import createEvent from "../../DataCollector/createEvent";
+import createEvent from "../../../core/createEvent";
```

Comment:
Let's please add a TODO to fix this.

Reply:
I'm working on an additional PR to fix this right now.

Reply:
For reference: https://jira.corp.adobe.com/browse/CORE-37059